crates/trippy-tui/locales/app.yml

```diff
@@ -3,439 +3,549 @@ trippy:
   en: "max-hosts"
   fr: "nombre d'hôtes maximum"
   tr: "maksimum hostlar"
+  it: "numero massimo di host"
 privacy:
   en: "privacy"
   fr: "confidentialité"
   tr: "gizlilik"
+  it: "privacy"
 privileged:
   en: "privileged"
   fr: "privilégié"
   tr: "ayrıcalıklı"
+  it: "privilegiato"
 unprivileged:
   en: "unprivileged"
   fr: "non privilégié"
   tr: "ayrıcalıksız"
+  it: "non privilegiato"
 na:
   en: "n/a"
   fr: "non disponible"
   tr: "yok"
+  it: "n/d"
 discovered:
   en: "discovered %{hop_count} hops"
   fr: "%{hop_count} sauts découverts"
   tr: "%{hop_count} atlanan keşfedildi"
+  it: "scoperti %{hop_count} salti"
 discovered_flows:
   en: "discovered %{hop_count} hops and %{flow_count} unique %{plural_flows}"
   fr: "%{hop_count} sauts et %{flow_count} %{plural_flows} uniques découverts"
   tr: "%{hop_count} atlanan ve %{flow_count} benzersiz %{plural_flows} keşfedildi"
+  it: "scoperti %{hop_count} salti e %{flow_count} %{plural_flows} unici"
 unknown:
   en: "unknown"
   fr: "inconnu"
   tr: "bilinmeyen"
+  it: "sconosciuto"
 icmp:
   en: "icmp"
   fr: "icmp"
   tr: "icmp"
+  it: "icmp"
 udp:
   en: "udp"
   fr: "udp"
   tr: "udp"
+  it: "udp"
 tcp:
   en: "tcp"
   fr: "tcp"
   tr: "tcp"
+  it: "tcp"
 status_failures:
   en: "%{failure_count} of %{total_probes} (%{failure_rate}%) probes failed"
   fr: "%{failure_count} sur %{total_probes} (%{failure_rate}%) sondes ont échoué"
   tr: "%{failure_count} / %{total_probes} (%{failure_rate}%) sondajın başarısız olması"
+  it: "%{failure_count} di %{total_probes} (%{failure_rate}%) prove fallite"
 status_failed:
   en: "Failed"
   fr: "Échec"
   tr: "Başarısız"
+  it: "Fallito"
 status_running:
   en: "Running"
   fr: "En cours"
   tr: "Çalışıyor"
+  it: "In esecuzione"
 status_frozen:
   en: "Frozen"
   fr: "Gelé"
   tr: "Dondurulmuş"
+  it: "Congelato"
 awaiting_data:
   en: "Awaiting data..."
   fr: "En attente de données..."
   tr: "Veri bekleniyor..."
+  it: "In attesa di dati..."
 header_help:
   en: "elp"
   fr: "Aide"
   tr: "Yardım"
+  it: "Aiuto"
 header_settings:
   en: "ettings"
   fr: "Paramètres"
   tr: "Ayarlar"
+  it: "Impostazioni"
 header_quit:
   en: "uit"
   fr: "Quitter"
   tr: "Çıkış"
+  it: "Uscita"
 title_hops:
   en: "Hops"
   fr: "Sauts"
   tr: "Atlananlar"
+  it: "Salti"
 title_frequency:
   en: "Frequency"
   fr: "Fréquence"
   tr: "Sıklık"
+  it: "Frequenza"
 title_samples:
   en: "Samples"
   fr: "Échantillons"
   tr: "Örnekler"
+  it: "Campioni"
 title_traces:
   en: "Traces"
   fr: "Traces"
   tr: "İzler"
+  it: "Tracce"
 title_flows:
   en: "Flows"
   fr: "Flux"
   tr: "Akışlar"
+  it: "Flussi"
 title_map:
   en: "Map"
   fr: "Carte"
   tr: "Harita"
+  it: "Mappa"
 title_help:
   en: "Help"
   fr: "Aide"
   tr: "Yardım"
+  it: "Aiuto"
 title_settings:
   en: "Settings"
   fr: "Paramètres"
   tr: "Ayarlar"
+  it: "Impostazioni"
 bsod_failed:
   en: "Trippy Failed :("
   fr: "Trippy a échoué :("
   tr: "Trippy Başarısız :("
+  it: "Trippy Fallito :("
```

Comment:
"Trippy Fallito :(" sounds like trippy failed in life. Here I think you wanna say that it failed executing? Or what is the context exactly?

Reply:
Ok, this is a bit funny, but in Italian I would either use "Trippy e' crashato" which is basically the italianization of "crashed" or something like "Trippy ha avuto un problema" ("Trippy encountered a problem")